wagtail/admin/templates/wagtailadmin/pages/listing/_modern_dropdown.html

```diff
@@ -3,7 +3,7 @@
 <div {{ self.attrs }} class="{{ classes|join:' ' }}" data-button-with-dropdown>
     <button class="{{ button_classes|join:' ' }}" data-button-with-dropdown-toggle data-hover-tooltip-content="{{ title }}" data-tippy-offset="[0, -2]">
         <span class="{% if hide_title %}w-sr-only{% endif %}">{{ title }}</span>
-        {% if icon_name %}{% icon name=icon_name classname='w-w-5 w-h-5' %}{% endif %}
+        {% if icon_name %}{% icon name=icon_name classname='w-w-4 w-h-4' %}{% endif %}
```

Comment:
It’d be nice to have better icon styling / sizing classes or attributes, but I decided to leave this out for now.